qiskit/visualization/latex.py

```diff
@@ -38,8 +36,8 @@ class QCircuitImage:
     Thanks to Eric Sabo for the initial implementation for Qiskit.
     """
 
-    def __init__(self, qubits, clbits, ops, scale, style=None,
-                 plot_barriers=True, reverse_bits=False, layout=None, initial_state=False,
```

Comment:
There were not used anyway, right? No need to deprecate-warning them?

Reply:
Correct. Just cleanup.